src/components/tooltip/tooltip.js

```diff
@@ -180,7 +196,8 @@ class Tooltip extends React.Component {
 
     const WrapperComponent = this.props.components.WrapperComponent || Wrapper;
     const BodyComponent = this.props.components.BodyComponent || Body;
-
+    const TooltipWrapper =
```

Comment:
Nitpick: to keep it consistent with the other variable names

- rename variable `TooltipWrapper` --> `TooltipWrapperComponent`
- rename component `TooltipWrapperComponent` --> `TooltipWrapper`

Reply:
Done 👍